README.md

```diff
@@ -1,15 +1,15 @@
-[![Docker Pulls](https://img.shields.io/docker/pulls/gnuton/vitasdk-docker.svg)](https://hub.docker.com/r/gnuton/vitasdk-docker)
+[![Docker Pulls](https://img.shields.io/docker/pulls/applelo/vitasdk-docker.svg)](https://hub.docker.com/r/applelo/vitasdk-docker)
```

Comment:
You can understand I cannot integrate anything from your repo on mine since it will break mine :D